### PR TITLE
after filters added to autocomplete will be removed from it automatic…

### LINKED ISSUE #300: after adding chips remove them from atuocomplete

### PR DESCRIPTION
**Pull request by hoseinmirian**
…ally

### CODE REVIEW

**Review by terazus — CHANGES_REQUESTED** [review]
If i understand correctly, this still doesn't work properly:
bugautocomplete
Can you clarify please?

**Comment by hoseinmirian** [comment]
> 
> 
> If i understand correctly, this still doesn't work properly:
> bugautocomplete
> Can you clarify please?

@terazus please ignore the test. just let me know if still has the issue

**Comment by terazus** [comment]
Seems to be fixed now.

File: src/components/Records/Search/Input/FilterAutocomplete.vue [hunk 1]
```diff
@@ -45,185 +45,187 @@
...
+     */
+    reset: function (selectedItem) {
+      selectedItem.filterSelected = {};
+    },
+    /**
+     * Clean up the given string by removing underscores and fills the stringsReplacement mapper variable.
+     * @param {String} string - the raw string to clean
+     * @returns {String} cleanedString - the string stripped of underscores.
+     */
+    cleanString: function (string) {
```
Comment (terazus) [inline src/components/Records/Search/Input/FilterAutocomplete.vue]: Dont we have a mixin/utils for that now?

**Review by terazus — CHANGES_REQUESTED** [review]
Change the cleanString to use from a mixin/util file.

File: src/components/Records/Search/Input/FilterAutocomplete.vue (hunk 1 above)
Comment (hoseinmirian) [reply src/components/Records/Search/Input/FilterAutocomplete.vue]: done

**Review by terazus — APPROVED** [review]

**Pull request merged**